### PR TITLE
I have added my name and username

### LINKED ISSUE #6: add Yussef to contributors file

### PR DESCRIPTION
**Pull request by YuUs01**
closes #6

### CODE REVIEW

File: contributors.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
 Contributors
 ============
 
-bananas24 - teacher
\ No newline at end of file
+bananas24 - teacher
+
```
Comment (bananas24) [inline contributors.md:5]: I don't like this blank line, remove it

File: contributors.md [hunk 2]
```diff
@@ -1,4 +1,6 @@
...
+Youssef - YuUs01
```
Comment (bananas24) [inline contributors.md:6]: All dashes must be in the same column

**Review by bananas24 — CHANGES_REQUESTED** [review]

File: contributors.md [hunk 3]
```diff
@@ -2,5 +2,4 @@ Contributors
 ============
 
 bananas24 - teacher
-
-Youssef - YuUs01
\ No newline at end of file
+Youssef - YuUs01
```
Comment (bananas24) [inline contributors.md:5]: Good job, now I would like to have all dashes in the same column.
For example:
``
bananas24 - teacher
``
``
Youssef   - YuUs01
``

**Review by bananas24 — CHANGES_REQUESTED** [review]
In the PR description, the word used is not one of the proper keywords. Check here:
https://docs.github.com/en/issues/tracking-your-work-with-issues/using-issues/linking-a-pull-request-to-an-issue

**Review by bananas24 — APPROVED** [review]
Thanks Youssef! Good job.
Next time, I want all commit messages, PR title and descriptionin English!

**Review by bananas24 — CHANGES_REQUESTED** [review]
Message to close the issue still missing from the PR. Otherwise, looking good.

**Review by bananas24 — APPROVED** [review]

**Review by Lowri-ui — APPROVED** [review]

**Review by tamasi17 — APPROVED** [review]
gg

**Review by ItsKein — APPROVED** [review]

**Comment by Karls3fni** [comment]
grande

**Pull request merged**